srtcore/core.cpp

```diff
@@ -11592,6 +11597,8 @@ bool srt::CUDT::runAcceptHook(CUDT *acore, const sockaddr* peer, const CHandShak
     acore->m_HSGroupType = gt;
 #endif
 
+    // Set the default value
+    acore->m_RejectReason = SRT_REJX_FALLBACK;
     try
     {
         int result = CALLBACK_CALL(m_cbAcceptHook, acore->m_SocketID, hs.m_iVersion, peer, target);
```

Comment:
The only way to alter it in the listener callback is by using `srt_setrejectreason`, and this function is only allowed to set the value above 1000 (`SRT_REJC_PREDEFINED`). Those values are allowed no matter if they are covered in `access_control.h` because this file contains only a proposed standard, but the application doesn't have to use these codes for itself.